code/modules/mob/living/living.dm

```diff
@@ -1,5 +1,5 @@
 /mob/living/proc/Life()
-	if(stat == DEAD || notransform) //If we're dead or notransform don't bother processing life
+	if(stat == DEAD || notransform || HAS_TRAIT(src, TRAIT_STASIS)) //If we're dead or notransform don't bother processing life
 		return
 
 	handle_status_effects() //all special effects, stun, knockdown, jitteryness, hallucination, sleeping, etc
```

Comment:
This lets you stunlock someone inside a stasis bag forever with no hope of getting out

Reply:
All incapacitating status effects are handled by datums, which aren't affected by this.

Reply:
what about stamina loss?

Reply:
It won't cause new disables while you're bagged and any existing ones will fall off.

Reply:
stamina loss will keep you stunned yes because its handled in status effects

Reply:
Stamloss's stun is from living's updatestamina. This is called by adjust_stamina, called by handle_staminaloss, called by handle_status_effects. If you're in stasis, this won't happen. Stamloss's paralyze is a status_effect datum, which as I said before will expire even in stasis.